logging/Limiting_Access_to_Logs.md

```diff
@@ -15,101 +26,172 @@ only for a specified namespace.
 
 ## Managing User Accounts for Restricted Access
 
-Use the `/logging/bin/user.sh` script to create or delete Kibana accounts 
-that have access to only the log messages and Kibana tenant space 
-associated with a specified namespace or SAS Viya tenant. You can use 
-the script to create user accounts for a Kibana tenant space anytime 
-after you have run the `onboard.sh` to implement access limitations for 
-the Kibana tenant space.
+### Introduction
+
+You can create or delete Kibana accounts 
+that have access to only the log messages and Kibana-tenant space 
+associated with a specified namespace or SAS Viya tenant. 
 
-This is the syntax for the script to add a user:
+### Creating User Accounts
+After you run the `onboard.sh` to implement access limitations for 
+the Kibana-tenant space, you can run 
+the  `/logging/bin/user.sh` script to create user accounts for a Kibana-tenant space. 
 
-```
-/logging/bin/user.sh CREATE --namespace <namespace>  [--tenant <tenant>]  [--user][<username>] [--password <password>]
-```
-The `namespace` parameter is required. It specifies the Kubernetes namespace to which the user should have access
 
-The `tenant` parameter specifies the SAS Viya tenant to which the user should have access.
+Here is the syntax for the script to add a user:
 
-If you do not specify a name for the `user` parameter, a user is 
+<pre>```
+/logging/bin/user.sh CREATE --namespace <i>namespace</i> [--tenant <i>tenant</i>] 
+[--user][<i>user_name</i>] [--password <i>password</i>]
+</pre>
+
+- *namespace* is required. It specifies the Kubernetes namespace to which the 
+  user can access.
+- *tenant* specifies the SAS Viya tenant to which the user can access.
+- *user* specifies a user name. If you do not specify a user name, a user is 
 created with the default name of `<Kibana tenant space name>_admin`. 
-For example, if you ran the script and specified `--namespace mynamespace --tenant mytenant1 --user`, the user would 
-be `mynamespace_mytenant1_admin`.
+For example, if you ran the script and specified 
+`--namespace mynamespace --tenant mytenant1 --user`, the user would be `mynamespace_mytenant1_admin`.
+
+You can create new users and link them to these access controls by running the 
+user.sh script.  For example, to create a user called "chen" and link the user to these 
+access controls, submit the following command:
 
-This is the syntax for the script to remove a user:
+<pre>
+/logging/bin/user.sh CREATE --user <i>user_name</i> --password <i>password</i> 
+--namespace _all_ --tenant _all_
+</pre>
 
-```
-/logging/bin/user.sh DELETE --user <username>
-```
+Where: 
+
+- *user_name* is "chen".
+- *password* is the password to sign in to Kibana as the user "chen".
+
+### Deleting a User Account
+
+Here is the syntax for the script to delete a user:
+
+<pre>
+/logging/bin/user.sh DELETE --user <i>user_name</i>
+</pre>
```

Comment:
See note above, re:paths.

Reply:
Applied.

Reply:
Sorry, looks like the leading slash has re-appeared.